### DISCUSSION TITLE
How to deal with Encoding decimal with exact precision what is been applied to the property

### DISCUSSION THREAD

**Discussion by vivekguptaraw** [davdroman/PreciseDecimal#5] [Q&A]
I am storing decimal value (0.5025) as attached in 1st screenshot.
But it is encoding value as 0.5024999999999999 as attached in 2nd screenshot.
Note - I dont want to encode value in String. Only Decimal with double is allowed.

<img width="219" alt="Screenshot 2022-01-20 at 8 04 16 PM" src="https://user-images.githubusercontent.com/9131749/150359130-7604b6c7-f931-41c0-b532-fca91fd76221.png">
<img width="258" alt="Screenshot 2022-01-20 at 8 03 25 PM" src="https://user-images.githubusercontent.com/9131749/150359139-dd8332a9-5e47-4fe9-801e-ee0411001794.png">

**Comment by davdroman** [comment]
Can you provide some example code of how you're currently doing the encoding? Thanks.
